src/flp/types/fixedpoint_l2.rs

```diff
@@ -2,7 +2,9 @@
 
 //! A [`Type`] for summing vectors of fixed point numbers where the
 //! [L2 norm](https://en.wikipedia.org/wiki/Norm_(mathematics)#Euclidean_norm)
-//! of each vector is bounded by `1`.
+//! of each vector is bounded by `1` and adding [discrete Gaussian
+//! noise](https://arxiv.org/abs/2004.00010) in order to achieve central
```

Comment:
Let's be consistent about "server-DP" or "central-DP".
```suggestion
//! noise](https://arxiv.org/abs/2004.00010) in order to achieve server
```

Reply:
I agree.